cloudnet-modules/cloudnet-signs/src/main/java/de/dytanic/cloudnet/ext/signs/bukkit/BukkitSignManagement.java

```diff
@@ -40,28 +42,27 @@
      * @deprecated SignManagement should be accessed via the {@link de.dytanic.cloudnet.common.registry.IServicesRegistry}
      */
     @Deprecated
+    @ApiStatus.ScheduledForRemoval
     public static BukkitSignManagement getInstance() {
         return (BukkitSignManagement) CloudNetDriver.getInstance().getServicesRegistry().getFirstService(AbstractSignManagement.class);
     }
 
-
     @Override
     protected void updateSignNext(@NotNull Sign sign, @NotNull SignLayout signLayout, @Nullable ServiceInfoSnapshot serviceInfoSnapshot) {
         Bukkit.getScheduler().runTask(this.plugin, () -> {
-            Location location = this.toLocation(sign.getWorldPosition());
-
-            if (location != null) {
-
-                Block block = location.getBlock();
-
-                if (block.getState() instanceof org.bukkit.block.Sign) {
-                    org.bukkit.block.Sign bukkitSign = (org.bukkit.block.Sign) block.getState();
-
-                    this.updateSign(location, sign, bukkitSign, signLayout, serviceInfoSnapshot);
+            final Location location = this.toLocation(sign.getWorldPosition());
```

Comment:
We don't use final for local variables at any place in this project, so why would we do this here? Imo final on local variables is pretty unnecessary.